app/helpers/subscriptions_management_helper.rb

```diff
@@ -0,0 +1,9 @@
+module SubscriptionsManagementHelper
+  def get_subscription_title(subscription)
+    subscription["subscriber_list"]["title"]
+  end
+
+  def get_page_heading(subscription)
+    "How often do you want to get emails about ‘#{get_subscription_title(subscription)}’?"
```

Comment:
I think this text should live in a locale file.  Locale entries can have variable interpolation, so the locale entry could be something like

```yaml
title: How often do you want to get emails about ‘%{subscription_title}’?
```

And then the view would use:

```ruby
page_heading = t("title", subscription_title: subscription["subscriber_list"]["title"])
```

Reply:
Oh, that's nice. I'll update